cg/store/models.py

```diff
@@ -728,7 +728,7 @@ class Invoice(Model):
     created_at = Column(types.DateTime, default=dt.datetime.now)
     updated_at = Column(types.DateTime, onupdate=dt.datetime.now)
     invoiced_at = Column(types.DateTime)
-    comment = Column(types.Text)
+    comment = Column(types.Text, nullable=False)
```

Comment:
I don't think comments are required always. So this column can be NULL

Reply:
I agree, they can be nullable